rest-api/src/test/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/DisplayApiControllerTest.java

```diff
@@ -74,4 +85,15 @@ public void getWizardPageDisplay() {
         subject.displayWizardPageStructureIdGet("TestAddressBookCase", "createCase");
         verify(displayService).findWizardPageForCaseType("TestAddressBookCase", "createCase");
     }
+    
+    @Test
+    public void getJurisdictionUiConfigs() {
+    	List<String> references = Collections.singletonList("AUTOTEST1");
+    	JurisdictionUiConfig jurisdictionUiConfig = new JurisdictionUiConfig();
+    	when(jurisdictionUiConfigService.getAll(any())).thenReturn(Collections.singletonList(jurisdictionUiConfig));
+    	JurisdictionUiConfigResult result = subject.getJurisdictionUiConfigs(Optional.of(references));
+    	verify(jurisdictionUiConfigService).getAll(references);
+    	assertAll(() -> assertEquals(1, result.getConfigs().size()),
+				() -> assertEquals(jurisdictionUiConfig, result.getConfigs().get(0)));
+    }
```

Comment:
More tests could be added. At least a test for the case of returning empty list should be included.